src/js/pm-image-editor.js

```diff
@@ -444,6 +448,11 @@
                     $scope.$broadcast('updateSelection', $scope.editorId, $scope.editor.selectionCss());
 
                     $scope.updateHistoryButtons();
+
+                    $scope.state = angular.copy($scope.editor.history.current());
+                    setTimeout(function() {
+                        $scope.$apply();
+                    }, 0);
```

Comment:
Because selection is set during directives initialization (as the first value). And this causing "$digest in progress" exception. After this (when user interacts with editor) no such issues are triggered.